executor/api/kafka/server.go

```diff
@@ -76,12 +80,24 @@ func NewKafkaServer(fullGraph bool, workers int, deploymentName, namespace, prot
 			return nil, fmt.Errorf("Unknown transport %s", transport)
 		}
 	}
+	sslKakfaServer := util.GetSslElements()
+	var producerConfigMap = kafka.ConfigMap{"bootstrap.servers": broker,
+		"go.delivery.reports": false, // Need this othewise will get memory leak
+	}
+	if broker != "" {
+		if util.KafkaSecurityProtocol == "ssl" {
```

Comment:
:thought_balloon: In `worker.go` there's some lower-casing going on.  It'd be good to be consistent about what we expect - either for the value to be used as-is (which generally seems the case in this area of Core) or to normalise the value